code/obj/item/cigarette.dm

```diff
@@ -517,12 +553,45 @@
 
 	New()
 		..()
-		src.reagents.maximum_volume = 600
```

Comment:
Botany herbs are likely to have way over 20 units when you roll them. If I am reading this right, wouldn't this change severely nerf custom joints?

Reply:
It shouldn't but I'll make sure, I'm pretty sure in the plant code it sets the maximum level to whatever's in the herb.

Reply:
It won't but seeing as all sources set the maximum volume I'll just revert it. Pretty much a useless change.